packages/testing-docs/docs/BE Tests/ Common /Networks navigation - Production/id565 - Networks navigation Prod - Common - Navigation to Sepolia.md

```diff
@@ -0,0 +1,17 @@
+---
+tags: ['Full test', 'manual', 'Navigation', 'Network', 'regression', 'To Automate', 'ZKF-2507', 'Active']
```

Comment:
@olehbairak Do we decide to remove all ZKF-**** tags?

Reply:
Yes, fixed.
Thank you!